src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -33,16 +32,24 @@ import _ from 'lodash';
 const MODULE_NAME = 'AuthorizationService';
 
 export default class AuthorizationService {
-  public static canPerfomAction(entity: AuthorizationActions, authAction: Action): boolean {
-    switch (authAction) {
+  public static canPerfomAction(authActions: any, action: Action): boolean {
```

Comment:
Avoid usage of any
use AuthorizationActions | SiteAuthorizationActions
SiteAuthorizationActions inherits from AuthorizationActions
Downcast to SiteAuthorizationActions to have access to the props in this method